<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scicalc</class>
 <widget class="QMainWindow" name="scicalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>scicalc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="ScicalcEdit" name="edit_input"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_refresh">
        <property name="text">
         <string>&amp;refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>168</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionLoad_recent_file_on_startup"/>
    <addaction name="actionAuto_save_on_close"/>
    <addaction name="actionGeneral_settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionReadme"/>
    <addaction name="actionConstants"/>
    <addaction name="actionFunctions"/>
    <addaction name="separator"/>
    <addaction name="actionChangelog"/>
    <addaction name="actionAbout_scicalc"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionLoad_recent_file_on_startup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>load recent file on startup</string>
   </property>
  </action>
  <action name="actionAuto_save_on_close">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>auto-save changes on close</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionReadme">
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
  <action name="actionConstants">
   <property name="text">
    <string>Constants</string>
   </property>
  </action>
  <action name="actionFunctions">
   <property name="text">
    <string>Functions</string>
   </property>
  </action>
  <action name="actionGeneral_settings">
   <property name="text">
    <string>general settings...</string>
   </property>
  </action>
  <action name="actionAbout_scicalc">
   <property name="text">
    <string>About scicalc</string>
   </property>
  </action>
  <action name="actionChangelog">
   <property name="text">
    <string>Changelog</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScicalcEdit</class>
   <extends>QTextEdit</extends>
   <header>scicalcedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
